data/migrations/24.lua

```diff
@@ -1,3 +1,7 @@
 function onUpdateDatabase()
-	return false
+	print("> Updating database to version 25 (Guildhall support)")
+	db.query("ALTER TABLE `guilds` ADD `house_id` int(11) NOT NULL DEFAULT '0'")
+	db.query("ALTER TABLE `houses` ADD `guild_id` int(11) NOT NULL DEFAULT '0'")
```

Comment:
A cross-reference on both ends is just looking for trouble. Which one wins when they contradict?
It's SQL, we should need just one. Can a house by owned by both "owner" and "guild_id"? If not, it's easier to catch that by using `guild_id` on the `houses` table.